module/modmanager/manager.go

```diff
@@ -87,15 +92,17 @@ type addedResource struct {
 
 // Manager is the root structure for the module system.
 type Manager struct {
-	mu                      sync.RWMutex
-	logger                  logging.Logger
-	modules                 map[string]*module
-	parentAddr              string
-	rMap                    map[resource.Name]*module
-	untrustedEnv            bool
-	removeOrphanedResources func(ctx context.Context, rNames []resource.Name)
-	restartCtx              context.Context
-	restartCtxCancel        context.CancelFunc
+	mu                        sync.RWMutex
+	logger                    logging.Logger
+	modules                   map[string]*module
+	parentAddr                string
+	rMap                      map[resource.Name]*module
+	untrustedEnv              bool
+	viamHomeDir               string
+	moduleDataParentDirectory string
```

Comment:
Probably want to align on using `Dir` or `Directory` for these adjacent variable names.

I'd probably vote for `Dir` here to have avoided the reflowing of every member in this structure, but the damage was already done -- I already scanned the diff line by line to convince myself these were the only two changes :)

Reply:
Oops -- thanks for catching the inconsistency.

Reply:
Renamed to `Dir`